index.js

```diff
@@ -137,11 +137,15 @@ const template = (data, name) => {
     <time>${new Date(data.created_at).toISOString()}</time>
   </metadata>
   <trk>
-    <name>${dayTime} ${name}</name>${
+    <name>${data.notes ? data.notes : `${dayTime} ${name}`}</name>${
+    data.description
+      ? `
+    <desc>${data.description}</desc>`
+      : ''
+    }${
     type
       ? `
-    <type>${type}</type>
-    `.trim()
```

Comment:
Yes but that also removes the starting new line, and the `<type>` tag ends up on the same line as the previous tag.
I actually just "manually trimmed" the line by removing the trailing new line : end of string character is just after the closing tag `</type>` instead of on the next line.

Reply:
Makes sense now, thanks!